pennylane/data/data_manager/__init__.py

```diff
@@ -16,28 +16,36 @@
 them.
 """
 
+import sys
 import urllib.parse
 from concurrent import futures
 from functools import lru_cache
 from pathlib import Path
 from time import sleep
-from typing import Iterable, Mapping, Optional, Union
-import sys
+from typing import Any, Iterable, Mapping, Optional, Union
+
 from requests import get, head
 
 from pennylane.data.base import Dataset
 from pennylane.data.base.hdf5 import open_hdf5_s3
 from pennylane.data.data_manager import progress
 
+from .graphql import (
+    _get_dataset_urls,
+    _get_parameter_tree,
+    list_data_names,
+    list_attributes,
+)
 from .foldermap import DataPath, FolderMapView, ParamArg
 from .params import DEFAULT, FULL, format_params
 
+
 S3_URL = "https://datasets.cloud.pennylane.ai/datasets/h5"
 FOLDERMAP_URL = f"{S3_URL}/foldermap.json"
 DATA_STRUCT_URL = f"{S3_URL}/data_struct.json"
 
 
-@lru_cache(maxsize=1)
+# @lru_cache(maxsize=1)
```

Comment:
Is this supposed to be uncommented or removed?

Reply:
Good catch :fishing_pole_and_fish: Uncommented thanks!